lib/govuk_navigation_helpers/content_item.rb

```diff
@@ -77,6 +77,34 @@ def curated_taxonomy_sidebar_links
       end
     end
 
+    def navigation_sidebar_related_links
```

Comment:
This method name isn't based on the key from the content-store response whereas the others do - is this by design?